src/main/java/view/DateTimeFormat.java

```diff
@@ -0,0 +1,18 @@
+package view;
+
+import java.time.format.DateTimeFormatter;
+
+public enum DateTimeFormat {
+    DATE(DateTimeFormatter.ofPattern("MM월 dd일 E요일")),
+    TIME(DateTimeFormatter.ofPattern("HH:mm"));
```

Comment:
```suggestion
    DATE("MM월 dd일 E요일"),
    TIME("HH:mm");
```
DateTimeFormatter 생성 호출이 반복되네요. 중복 코드는 줄이기 위해 위와 같이 수정하는 것은 어려울까요?

Reply:
멤버 변수로 String인 `dateTimePattern`과 DateTimeFormatter인 `dateTimeFormatter`로 재구성 하여 중복 코드를 줄이고 반복적으로 DateTimeFormatter가 생성되는 것을 방지하였습니다. 제안하신 방법을 적용하여 중복된 코드를 줄일 수 있었습니다!

```java
public enum DateTimeFormat {
    DATE("MM월 dd일 E요일"),
    TIME("HH:mm");

    private final String dateTimePattern;
    private final DateTimeFormatter dateTimeFormatter;

    DateTimeFormat(String dateTimePattern) {
        this.dateTimePattern = dateTimePattern;
        this.dateTimeFormatter = createDateTimeFormatter();
    }

    public DateTimeFormatter createDateTimeFormatter() {
        return DateTimeFormatter.ofPattern(dateTimePattern);
    }

    public DateTimeFormatter getDateTimeFormatter() {
        return dateTimeFormatter;
    }
}
```